participants/anujji1999.md

```diff
@@ -1,5 +1,10 @@
-Name : Anuj Gupta
-Github Username : anujji1999
+```Name : Anuj Gupta```
```

Comment:
You don't need three of these ``` for a single line. Usually you use triple of those only when you want it to cover multiple lines. This would work the same if you do `Name: Anuj Gupta`. You can see the markdown in the message by going to the edit button of this message.